README.md

```diff
@@ -120,11 +120,51 @@ luatest.helpers.retrying({timeout = 1, delay = 0.1}, failing_function, arg1, arg
 luatest.helpers.retrying({}, function() server:http_request('get', '/status') end)
 ```
 
+## List of luatest functions
+
+| Assertiions |  |
+| :--- | --- |
+| `assert (value[, message])` | Check that value is truthy. |
+| `assert_almost_equals (actual, expected, margin[, message])` | Check that two floats are close by margin. |
+| `assert_covers (actual, expected[, message])` | Checks that map contains the other one. |
+| `assert_equals (actual, expected[, message[, deep_analysis]])` | Check that two values are equal. |
+| `assert_error (fn, ...)` | Check that calling fn raises an error. |
+| `assert_error_msg_contains (expected_partial, fn, ...)` | |
+| `assert_error_msg_content_equals (expected, fn, ...)` | Strips location info from message text. |
+| `assert_error_msg_equals (expected, fn, ...)` | Checks full error: location and text. |
+| `assert_error_msg_matches (pattern, fn, ...)` | |
+| `assert_eval_to_false (value[, message])` | Alias for assert_not. |
+| `assert_eval_to_true (value[, message])` | Alias for assert. |
+| `assert_is (actual, expected[, message])` | Check that values are the same. |
+| `assert_items_equals (actual, expected[, message])` | Check that the items of table expected are contained in table actual. |
+| `assert_nan (value[, message])` | |
+| `assert_not (value[, message])` | Check that value is falsy. |
+| `assert_not_almost_equals (actual, expected, margin[, message])` | Check that two floats are not close by margin. |
+| `assert_not_covers (actual, expected[, message])` | Checks that map does not contain the other one. |
+| `assert_not_equals (actual, expected[, message])` | Check that two values are not equal. |
+| `assert_not_is (actual, expected[, message])` | Check that values are not the same. |
+| `assert_not_nan (value[, message])` | |
+| `assert_not_str_contains (actual, expected[, is_pattern[, message]])` | Case-sensitive strings comparison. |
+| `assert_not_str_icontains (value, expected[, message])` | Case-insensitive strings comparison. |
+| `assert_str_contains (value, expected[, is_pattern[, message]])` | Case-sensitive strings comparison. |
+| `assert_str_icontains (value, expected[, message])` | Case-insensitive strings comparison. |
+| `assert_str_matches (value, pattern[, start=1[, final=value:len()[, message]]])` | Verify a full match for the string. |
+| **Flow control** |  |
+| `fail (message)` | Stops a test due to a failure. |
+| `fail_if (condition, message)` | Stops a test due to a failure if condition is met. |
+| `skip (message)` | Skip a running test. |
+| `skip_if (condition, message)` | Skip a running test if condition is met. |
+| `success ()` | Stops a test with a success. |
+| `success_if (condition)` | Stops a test with a success if condition is met. |
+| **Suite and groups** |  |
+| `after_suite (fn)` | Add after suite hook. |
+| `before_suite (fn)` | Add before suite hook. |
+| `group (name)` | Create group of tests. |
+
```

Comment:
Please add an example with any of this asserts like:
```lua
local t = require('luatest')
...

t.assertEquals(...)
```

Because I don't understand where is the holder of this asserts

Reply:
ok. please don't use camelCase in your projects. one day it will be removed

Reply:
BTW. isn't it already in usage?

Reply:
> BTW. isn't it already in usage?

Please put this section after the main usage code block